actions/helm-setup/action.yaml

```diff
@@ -0,0 +1,39 @@
+name: "Setup Helm"
+description: "Setup everything necessary to deploy or destroy Helm charts"
+inputs:
+  python-version:
+    description: "Python version"
+    default: "3.10"
+    required: false
+  kubectl-version:
+    description: "Kubectl version"
+    default: "v1.23.0"
+    required: false
+  helm-version:
+    description: "Helm version"
+    default: "v3.10.1"
+    required: false
+runs:
+  using: "composite"
+  steps:
+    # TODO: replace by ci runner image
+    - name: Setup python
+      uses: actions/setup-python@v1
```

Comment:
what does this action need Python for?

Reply:
I think GCloud needs it, I will check again to make sure

Reply:
It is not needed, I removed it